.github/workflows/test.yml

```diff
@@ -6,7 +6,6 @@ on:
 
 jobs:
   industrial_ci:
-    if: github.event.pull_request.draft == false
```

Comment:
Is that on purpose? I would assume yes and it's fine, but I just want to be sure you are aware of it.

Reply:
Yes. That line doesn't make sense for me, as I want to see if my CI is failing before I request a review. It is already deleted in the master branch (I will master to develop later).